rules/config

```diff
@@ -202,3 +202,5 @@ BUILD_MULTIASIC_KVM = n
 
 # INCLUDE_MUX - build docker-mux for dual ToR (Gemini)
 INCLUDE_MUX = y
+
+STORAGE_PUBLIC_TOKEN = ?st=2021-11-02T13%3A34%3A45Z&se=2030-11-03T13%3A34%3A00Z&sp=r&sv=2018-03-28&sr=c&sig=7MmsVW2EUGYSUr94IPJGBxwI1qYI0VQx%2Fa7z0koJT60%3D
```

Comment:
is there plan to remove the this token? it does not seem that the token is useful.

Reply:
@lguohan , do you mean to publish a storage account public accessible? so we do not need to token at all?